mkdocs.yml

```diff
@@ -55,13 +55,13 @@ pages:
   - Puppet Gluster: Administrator Guide/Puppet.md
   - RDMA Transport: Administrator Guide/RDMA Transport.md
   - GlusterFS iSCSI: Administrator Guide/GlusterFS iSCSI.md
-  - Configuring NFS-Ganesha server: Administrator Guide/NFS-Ganesha GlusterFS Intergration.md
+  - Configuring NFS-Ganesha server: Administrator Guide/NFS-Ganesha GlusterFS Integration.md
```

Comment:
In my commit I did rename the file. It shows up in my git commit message but I am not sure why its not showing up in this pull request.